src/Dev/FunctionalTest.php

```diff
@@ -134,7 +134,6 @@ protected function withBaseFolder($folder, $callback)
 
     /**
      * Submit a get request
-     * @uses Director::test()
```

Comment:
As far as I can tell `@uses` was for code-coverage

I'm not sure what the replacement for `@uses` is https://github.com/sebastianbergmann/phpunit/issues/4502

This was already not-recommended as `@uses ClassName::methodName (not recommended because too fine-grained)`

We're not doing any sort of code-coverage at this point

I did try using `#[UsesMethod('Director', 'test')]` however it will complain `Attribute "PHPUnit\Framework\Attributes\UsesMethod" cannot target method (allowed targets: class)`

I've elected to just remove these instead